pkg/network/ovn_kubernetes_test.go

```diff
@@ -344,3 +481,19 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 	g.Expect(errs[0]).To(MatchError("cannot change ovn-kubernetes MTU"))
 	g.Expect(errs[1]).To(MatchError("cannot change ovn-kubernetes genevePort"))
 }
+
+func extractOVNKubeConfig(g *WithT, objs []*uns.Unstructured) string {
+	for _, obj := range objs {
+		if obj.GetKind() == "ConfigMap" && obj.GetName() == "ovnkube-config" {
+			val, ok, err := uns.NestedString(obj.Object, "data", "ovnkube.conf")
+			g.Expect(err).NotTo(HaveOccurred())
+			g.Expect(ok).To(BeTrue())
+			return string(val)
+		}
+	}
+	return ""
+}
+
+func ptrToUINT32(x uint32) *uint32 {
```

Comment:
should be `ptrToUint32` (eg, like `strconv.ParseUint`)

Reply:
done